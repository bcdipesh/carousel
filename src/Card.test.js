import { render } from "@testing-library/react";
import Card from "./Card";

// Smoke test
it("renders Card component", function () {
  render(
    <Card
      caption="Lorem Ipsum Caption"
      src="lorem-ipsum"
      currNum={1}
      totalNum={2}
    />
  );
});

// Snapshot test
it("matches snapshot", function () {
  const { asFragment } = render(
    <Card
      caption="Lorem Ipsum Caption"
      src="lorem-ipsum"
      currNum={1}
      totalNum={2}
    />
  );
  expect(asFragment()).toMatchSnapshot();
});
