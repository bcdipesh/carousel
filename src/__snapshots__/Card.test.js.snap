// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      Lorem Ipsum Caption
    </h4>
    <img
      alt="Lorem Ipsum Caption"
      class="Card-image"
      src="lorem-ipsum"
    />
    <small
      class="Card-small"
    >
      Image 1 of 2.
    </small>
  </div>
</DocumentFragment>
`;
